docs/bluebear.md

```diff
@@ -0,0 +1,181 @@
+# nf-core/configs: BlueBEAR HPC Configuration
+
+Configuration for the BlueBEAR high performance computing (HPC) cluster at the University of Birmingham.
+
+## Setup
+
+To run a Nextflow pipeline, load the following modules,
+
+```bash
+module load bear-apps/2022b
+module load Nextflow/24.04.2
+```
+
+## Apptainer/Singularity
+
+BlueBEAR comes with Apptainer installed for running processes inside containers. Our configuration makes use of this
+when executing processes.
+
+We advise you create a directory in which to cache images using the following environment variable. For example,
+
+```bash
+export NXF_SINGULARITY_CACHEDIR="/rds/projects/_initial_/_project_/.apptainer"
+```
+
+where `_project_` is your project code and `_initial_` is its initial letter.
+
+You may notice that our nf-core configuration file enables Singularity, not Apptainer, for use with Nextflow. Due to
+their similarities (see [this announcement](https://apptainer.org/news/community-announcement-20211130/)), Apptainer
+creates the alias `singularity` for `apptainer` allowing both commands to be used. Nextflow's Singularity engine makes
+use of pre-built Singularity images, whereas its Apptainer engine does not.
+
+## Run
+
+**Do not** run Nextflow pipelines on a login node. Instead, create a submission script or start an interactive job.
+
+To run a Nextflow pipeline, specify the BlueBEAR profile option. For example,
+
+```bash
+nextflow run nf-core/_pipeline_ -profile bluebear --outdir results
+```
+
+where `_pipeline_` is the name of the pipeline.
+
+Make sure the job time requested is sufficient for running the full pipeline. If the job ends before the pipeline is
+complete, rerun it from a checkpoint using the `-resume` option. For example, run
+
+```bash
+nextflow run nf-core/_pipeline_ -profile bluebear --outdir results -resume
+```
+
+in a new job with more time requested.
+
+## Example
+
+Here is an example job submission script. This job runs the `nf-core/rnaseq` pipeline tests with the BlueBEAR config
+file. We request 1 hour to run the pipeline on 1 node with 2 tasks.
+
+```bash
+#!/bin/bash
+#SBATCH --account _project_
+#SBATCH --qos bbdefault
+#SBATCH --time 1:0:0
+#SBATCH --nodes 1
+#SBATCH --ntasks 2
+
+set -e
+
+module purge; module load bluebear
+module load bear-apps/2022b
+module load Nextflow/24.04.2
+
+# Directory in which to cache apptainer/singularity images
+export NXF_SINGULARITY_CACHEDIR="/rds/projects/_initial_/_project_/.apptainer"
+
+nextflow run nf-core/rnaseq -profile bluebear,test --outdir test_results
+```
+
+## Clean work directory
+
+Nextflow caches pipeline files in a work directory (default is `work`). This is useful if you need to resume a job
+after a failure (using the `-resume` option). However, the `work` directory can fill up quickly. Regularly cleaning the
+work directory avoids filling up your project quota.
```

Comment:
You could consider by default adding the `cleanup = true` [^1] option to have the emptying of the `work/` happen by default, and to override this use the `debug` profile in addition to the institutional profile.

For examples: https://github.com/search?q=repo%3Anf-core%2Fconfigs%20cleanup%20%3D%20true&type=code

[^1]: Docs: https://www.nextflow.io/docs/latest/config.html#miscellaneous

Reply:
I was a little concerned about the warning in the Nextflow docs about this. It suggests `cleanup = true` prevents use of `-resume`. If cleanup only occurs on a successful run, is `-resume` still possible after a failed run?

Reply:
It only prevents `-resume` if the pipeline completes with exit code 0 (so no errors).

If the pipeline hits an error (exit code >1), all the files in `work` are retained and will not be deleted

The only case that it might be problematic is if someone is playing with parameters in later stages of a pipeline, and want to resume from midway but with a different parameter, yes in this case this `resume` would not work (if that makes sense)

Reply:
Ah fantastic, I misunderstood the docs slightly. This sounds like a good addition, thanks!